### PR TITLE
Filter all widgets when zooming on the volumetry

### PR DESCRIPTION
**Pull request by martinratinaud**
As asked by Henri and users at "Santé"

### CODE REVIEW

**Review by martinratinaud — COMMENTED** [review]
@clementbiron please take a look even though I merge it now

**Review by clementbiron — COMMENTED** [review]
LGTM

File: src/modules/Common/pages/hashtags/[hashtag].tsx [hunk 1]
```diff
@@ -170,6 +171,9 @@ const HashtagPage = ({
         <div className="rf-grid-row">
           <div className="rf-col">
             <div className="text-center rf-myw">
+              <Link href="/">
```
Comment (clementbiron) [inline src/modules/Common/pages/hashtags/[hashtag].tsx:176]: Should be a breadcrumb no ?

Comment (martinratinaud) [reply src/modules/Common/pages/hashtags/[hashtag].tsx:176]: Yes, please add it in your next or 😍

**Pull request merged**